src/solver/solve.jl

```diff
@@ -21,12 +21,18 @@ walltime_in_days(es::EfficiencyStats) = es.walltime * (1 / (24 * 3600)) #=second
 
 function timed_solve!(integrator)
     device = ClimaComms.device(integrator.u.c)
+    comms_ctx = ClimaComms.context(device)
     local sol
+    original_stdout = stdout
+    if !ClimaComms.iamroot(comms_ctx)
+        redirect_stdout(devnull)
+    end
     @time "solve!:" begin
         walltime = ClimaComms.elapsed(device) do
```

Comment:
`walltime` here already contains the time to solution. You can just use `@info` or any other logging statement to print it alongside all the other logging information using the same logger, instead of treating this one in a special way